### PR TITLE
Apply tslint to services.ts file

### PR DESCRIPTION
**Pull request by yuit**
Slowly applying lint rule to services folder as doing all at once will be quite insane :sweat_smile:

### CODE REVIEW

File: Jakefile.js [hunk 1]
```diff
@@ -924,7 +924,8 @@ function lintFileAsync(options, path, cb) {
 var lintTargets = compilerSources
     .concat(harnessCoreSources)
     .concat(serverCoreSources)
-    .concat(scriptSources);
+    .concat(scriptSources)
+    .concat([path.join(servicesDirectory,"services.ts")]);
```
Comment (DanielRosenwasser) [inline Jakefile.js]: Space after ","

**Comment by DanielRosenwasser** [comment]
GitHub really sucks here.

**Comment by DanielRosenwasser** [comment]
At least we have gist: https://gist.github.com/DanielRosenwasser/03a80fe85cddf40201d9

**Comment by DanielRosenwasser** [comment]
There are a few places where you've touched the comments and switched `let` to `const`. That wasn't really necessary, and in the future, we should try to avoid that since it changes the last person associated with that line.

Anyhow, that's not all that big of a deal, so :+1:

**Comment by yuit** [comment]
@DanielRosenwasser  yeah. the list of lint errors are too big so I end up find-replace :sweat_smile:

**Pull request merged**